packages/headless/src/controllers/triggers/headless-query-trigger.ts

```diff
@@ -13,7 +13,9 @@ import {loadReducerError} from '../../utils/errors';
 import {buildController, Controller} from '../controller/headless-controller';
 
 /**
- * The `QueryTrigger` controller handles query triggers.
+ * The `Query` triggers allows the user to perform a new query. The `QueryTrigger` controller handles query triggers.
```

Comment:
It doesn't "allow the user" to perform a new query. If the query trigger rule is... triggered... and a `QueryTrigger` controller handles it, then that query _will_ be executed, and the user will just have to deal with it 😁